src/js/common/schemaform/Address.jsx

```diff
@@ -4,12 +4,13 @@ import classNames from 'classnames';
 import { set, assign } from 'lodash/fp';
 
 import { getDefaultFormState } from 'react-jsonschema-form/lib/utils';
+import commonDefinitions from 'vets-json-schema/dist/definitions.json';
 
 import * as address from './definitions/address';
 import { states } from '../utils/options-for-select';
 import { pureWithDeepEquals } from './helpers';
 
-const defaultStateSchema = address.schema().properties.state;
+const defaultStateSchema = address.schema({ definitions: commonDefinitions }).properties.state;
```

Comment:
We've already been pulling it in in many of the common definitions, so it's not much of a change from before. It's here because I want to be able to generate the different state lists for any instance of the component to switch between, since that's more efficient. It's tricky to do that based on the schema that's passed in to a particular instance of this component.

I would prefer not to have to do this, though, and be able to remove the common definitions file from our bundle. I think the best option for doing that is converting the address definition to use updateSchema instead of a custom field component (which I think is possible now).